src/compiler/types.ts

```diff
@@ -8235,12 +8235,13 @@ export const enum OuterExpressionKinds {
     Parentheses = 1 << 0,
     TypeAssertions = 1 << 1,
     NonNullAssertions = 1 << 2,
-    PartiallyEmittedExpressions = 1 << 3,
+    SatisfiesExpressions = 1 << 3,

```

Comment:
a new kind here makes sense to me 👍 it's clearer this way because `satisfies` isn't usually called an assertion

Reply:
Yeah though I'm not sure why we're pushing `PartiallyEmittedExpressions` up here. I guess it's consistent with previous values but it doesn't seem strictly required.